pkg/evaluator/evaluator_test.go

```diff
@@ -168,6 +168,22 @@ while true
     end
     continue = false
 end
+`, `
+while true
+    while true
+	break
```

Comment:
Needs one more indent?

Reply:
Ah, it was a tab, so it looked ok to me, but it looked 4 spaces to you. This language within language stuff is annoying.